Chapter19/19-1/db_tests.erl

```diff
@@ -0,0 +1,91 @@
+-module(db_tests).
+-import(db, [new/0, destroy/1, write/3, delete/2, read/2, match/2]).
+
+% -define(NOTEST, true).
+-include_lib("eunit/include/eunit.hrl").
+
+new_test() ->
+  ?assertEqual( [] , new() ).
+
+destroy_test() ->
+  ?assertEqual( ok , destroy([{1,2},{3,4},{5,6}]) ).
+
+write_test_() ->
+  [?_assertMatch([{1,     element}], write(1,     element, []) ),
```

Comment:
it's better to use assertEqual/2 if value on the left hand side is known. assertEqual/2 prints out better report in that case if something has failed.